src/TypeCatalogGen/TypeCatalogGen.cs

```diff
@@ -445,7 +443,8 @@ internal partial class PowerShellAssemblyLoadContext
             StringBuilder sourceCode = new StringBuilder(string.Format(CultureInfo.InvariantCulture, SourceHead, typeNameToAssemblyMap.Count));
             foreach (KeyValuePair<string, TypeMetadata> pair in typeNameToAssemblyMap)
             {
-                sourceCode.AppendLine(string.Format(CultureInfo.InvariantCulture, SourceFormat, pair.Key, pair.Value.AssemblyName));
+                sourceCode.AppendLine();
+                sourceCode.Append(string.Format(CultureInfo.InvariantCulture, SourceFormat, pair.Key, pair.Value.AssemblyName));
```

Comment:
We could move the sourceCode.AppendLine() before the cycle and use AppendLine() in 447 line.
SourceEnd below would be on new line too.

Reply:
I'm doing this because I don't want a trailing new line at the end.

Reply:
Updated a little to use only one `Append` call.